src/docs/eviction.dox

```diff
@@ -0,0 +1,103 @@
+/*! @m_page{{c,java},eviction, Eviction}
+
+All the operations performed in WiredTiger are on the data read into
+reconfigured amount of memory. WiredTiger uses memory as a cache for
+all the data on the disk and the data in memory forms the current
+working set.
+
+Since memory is limited and can't hold all the data stored on the disk,
+WiredTiger has to continuously move data that is currently not being
+accessed out of the memory to free up enough space to read data that
+are requested by the user but currently reside on the disk back into
+memory. This process is called eviction.
+
+@section overview Eviction overview
+
+The WiredTiger eviction runs in the background with one eviction server
+thread and several eviction worker threads.
+
+1) The eviction server thread walks the btrees and put the pages into
```

Comment:
put -> puts
<or>
put the pages into -> adds the pages to

Reply:
Does the eviction server thread put all the pages into the queue or only certain ones?

Reply:
I'm not sure on that.

Reply:
Added some details. I think we prefer to add the least recently used pages to the queue and each eviction queue has a size limit.